lib/webvalve/fake_service_config.rb

```diff
@@ -26,9 +26,9 @@ def service_url
 
     def missing_url_message
       <<~MESSAGE
-        There is no URL defined for #{service.name}.
+        There is no URL defined for #{service_class_name}.
         Configure one by setting the ENV variable "#{service_name.to_s.upcase}_API_URL"
-        or by using WebValve.register #{service.name}, url: "http://something.dev"
+        or by using WebValve.register "#{service_class_name}", url: "http://something.dev"
```

Comment:
im a little late to the party with this nit but should this message have the evaluated service_class_name in quotes so no one gets confused and uses the now deprecated way of registering a service?

Reply:
I guess that's handled globally in the manager class

Reply:
it's in quotes here, no? since this string is inside a heredoc the quotes are literals